lib/landable/traffic/user_tracker.rb

```diff
@@ -39,7 +39,8 @@ def record_page_view
       end
 
       def save
-        record_page_view
+        p = record_page_view
+        EventPublisher.new(p).publish
```

Comment:
I see the ```EventPublisher``` class handles whether hutch is enabled and connected.  Does it make sense to move that code here? 

so this would become something like
```EventPublisher.new(p).publish if hutch_enabled?```